api/valor_api/exceptions.py

```diff
@@ -397,6 +397,14 @@ class InvalidLLMResponseError(Exception):
     pass
 
 
+class BadValueInTestLLMClientsError(Exception):
```

Comment:
Is this error only used for testing? If so, Im not sure it should be included in the production code.